### ISSUE TITLE
Verify name test case 558

### ISSUE THREAD

**Issue by joanmarie** [WhatSock/w3c-alternative-text-computation#9]
Name test case 558 says:

```
if given
  <input type="text" value="peanuts" id="test">
  <img aria-labelledby="test" src="foo.jpg"/>
then the accessible name of the element with id of "test" is "peanuts"
```
And [this expectation is confirmed by the WhatSock tool](https://whatsock.github.io/w3c-alternative-text-computation/Autogenerated%20AccName%201.1%20Testable%20Statements%20-%20W3C/Name%20test%20case%20558.html).

But I don't think this is correct. As I [originally posted on the ARIA mailing list](https://lists.w3.org/Archives/Public/public-aria/2018Mar/0044.html):

The image is labeled by the input. But the only thing exposed by the input is the value attribute. So calculating the name from the input beginning with step 2 (i.e. current node is the input), we get:

* 2A doesn't apply: The input is not hidden
* 2B doesn't apply: Calculating a name; the input lacks aria-labelledby
* 2C doesn't apply: Calculating a name; the input lacks aria-label
* 2D doesn't apply imho: I don't see value as an attribute that defines a text alternative. And when I look at the name calculation stuff in HTML AAM, it doesn't say value is such an attribute. See section 5.1.1.
* 2E doesn't apply: The input is not embedded within the label *for another widget* (emphasis added). So while the input is referenced by aria-labelledby, I think that's trumped by the fact that what references it is an image rather than a widget.
* 2F doesn't apply: Input/textbox doesn't allow name from content.
* 2G doesn't apply: Input is not a text node.
* 2H doesn't apply: The input is not a descendant of an element whose name is being computed.
* 2I doesn't apply: The input lacks a tooltip attribute.

End of options. Empty string. And that's what the implementations say too.

**Comment by joanmarie** [comment]
Test case 560 is quite similar so adding it here. The only difference is that the image has an aria-label, but as with 558, it's the input whose name we're calculating; not the image.

**Comment by joanmarie** [comment]
This has been addressed as Bryan described on the mailing list: https://lists.w3.org/Archives/Public/public-aria/2018Mar/0050.html

I've updated the wiki. Once the other issues have been resolved, then I'll regenerate all the tests for WPT.

Closing this issue as fixed.
